Source/MQTTnet/Client/Options/MqttClientTlsOptionsBuilder.cs

```diff
@@ -132,5 +132,13 @@ public MqttClientTlsOptionsBuilder WithCipherSuitesPolicy(EncryptionPolicy encry
             return this;
         }
 #endif
+
+#if NET7_0_OR_GREATER
+        public MqttClientTlsOptionsBuilder WithCertificationAuthoritiesFile(string pemFile)
```

Comment:
another option will be `WithCaFile` is shorter -not as descriptive as the other methods in this class - but is aligned with other mqtt tooling.

Reply:
The other methods and properties in this project usually take the "long" version or adopt the same name when it is simply mapped to a property in the .NET framework.

Reply:
this comment is outdated, the last commit changed the signature to: `public MqttClientTlsOptionsBuilder WithTrustChain(X509Certificate2Collection chain)`

To align with .NET api this could be `.WithCustomTrustChain`